src/Raven.Server/Web/System/AdminMetricsHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="AdminMetricsHandler.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+using System.Threading.Tasks;
+using Raven.Database.Util;
+using Raven.Server.Routing;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.System
+{
+    public class AdminMetricsHandler : RequestHandler
+    {
+        [RavenAction("/admin/rootMetrics", "GET")]
```

Comment:
Feel free to change this, I just moved it away from AdminDatabasesHandler which isn't the right place for this action